handler/engine/src/test/java/io/knotx/fragments/engine/TaskEngineSingleOperationTest.java

```diff
@@ -274,6 +276,48 @@ void expectErrorAndSuccessEventLogEntries(VertxTestContext testContext, Vertx ve
         ));
   }
 
+  @Test
+  @DisplayName("Expect node debug in log event log entries when success transition handled.")
```

Comment:
Can you please add some mock behaviour and prepare test for it.
- behaviour that does 3 retries (failing action)

Expected result:
node log : behaviour log + [first invovation, second invocation, third invocation]